dev/docker/kerberos-hive/krb5formac.conf

```diff
@@ -0,0 +1,26 @@
+############################################
+# This file is used to configure the Kerberos client on the Mac OS. For example,
+# HDFS client need to use this file to connect to the Kerberos server. Code as below:
+# System.setProperty("java.security.krb5.conf", "/tmp/krb5.conf");  and the file '/tmp/krb5.conf'
+# is the content of this file.
```

Comment:
One left issue, can this be used for Linux. If so, you'd better change to a better file name.